Modules/Libs/QuestieLib.lua

```diff
@@ -590,6 +573,26 @@ objectiveFontString:Hide()
 ---@param desiredWidth number @Set the desired width to wrap, default: 275
 ---@return table[] @A table of wrapped lines
 function QuestieLib:TextWrap(line, prefix, combineTrailing, splitOnDot, desiredWidth, questid)
+    if not objectiveFontString then        
+        if _G["QuestLogObjectivesText"] then -- classic
+            frameObject = _G["QuestLogObjectivesText"] 
+        elseif _G["QuestInfoObjectivesText"] then -- Wotlk Classic
+            frameObject = _G["QuestInfoObjectivesText"]
+        end
```

Comment:
In most other files we do these kind of things in the very first lines of a fine and just concat them with `or` to get the last non nil one.

Like `local frameObject = QuestLogObjectivesText or QuestInfoObjectivesText`. Might be applicable here as well?

Reply:
Yeah you are totally right!

Reply:
Well maybe we can use a better name than `frameObject` which better reflects that we are referencing a default UI Frame?